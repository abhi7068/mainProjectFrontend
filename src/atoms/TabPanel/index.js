import Typography from '@material-ui/core/Typography';
import Box from '@material-ui/core/Box';
import React from 'react';
import PropTypes from 'prop-types';
// import { PROFILE_IMAGE_URL } from '../../constants/PROFILE_IMAGE_URL';
// import Avatar from '@material-ui/core/Avatar';
// import Typography from '@material-ui/core/Typography';

function TabPanel(props) {
  const { children, value, index, ...other } = props;

  return (
    <Typography
      component="div"
      role="tabpanel"
      hidden={value !== index}
      id={`vertical-tabpanel-${index}`}
      aria-labelledby={`vertical-tab-${index}`}
      {...other}
    >
      {value === index && <Box p={4}>{children} </Box>}
    </Typography>
  );
}
TabPanel.propTypes = {
  children: PropTypes.node,
  index: PropTypes.any.isRequired,
  value: PropTypes.any.isRequired,
};
export default TabPanel;
