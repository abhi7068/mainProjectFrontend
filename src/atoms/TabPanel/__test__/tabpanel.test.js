import React from 'react';
import TabPanel from '../index';
import { render } from '@testing-library/react';

it('test', () => {
  render(<TabPanel>help</TabPanel>);
});

it('snapshot', () => {
  const { asFragment } = render(<TabPanel />);

  expect(asFragment(<TabPanel />)).toMatchSnapshot();
});
