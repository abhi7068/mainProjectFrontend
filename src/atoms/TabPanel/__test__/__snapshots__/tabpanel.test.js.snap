// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<DocumentFragment>
  <div
    aria-labelledby="vertical-tab-undefined"
    class="MuiTypography-root MuiTypography-body1"
    id="vertical-tabpanel-undefined"
    role="tabpanel"
  >
    <div
      class="MuiBox-root MuiBox-root-64"
    >
       
    </div>
  </div>
</DocumentFragment>
`;
